app/Views/errors/html/debug.css

```diff
@@ -65,13 +65,10 @@ p.lead {
     display: inline;
 }
 
-.footer {
+.environment {
     background: var(--dark-bg-color);
     color: var(--light-text-color);
-}
-.footer .container {
     border-top: 1px solid #e7e7e7;
```

Comment:
```suggestion
```
I think we don't need it.

![Screenshot 2024-03-15 043000](https://github.com/codeigniter4/CodeIgniter4/assets/9530214/a7c876bb-d5ef-43f6-b5a4-8439269e9752)

Reply:
Okay. Removed.